drivers/serial/uart_stm32.c

```diff
@@ -1786,16 +1786,24 @@ static int uart_stm32_pm_action(const struct device *dev,
 		/* Stop device clock. Note: fixed clocks are not handled yet. */
 		err = clock_control_off(data->clock, (clock_control_subsys_t)&config->pclken[0]);
 		if (err != 0) {
-			LOG_ERR("Could not enable (LP)UART clock");
+			/*
+			 * Warn without console output when going to sleep,
+			 * "Could not enable (LP)UART clock"
+			 * but don't block PM suspend
+			 */
```

Comment:
Not being able to clock off is an error. Logging it is justified and won't have negative side effect (an error is already returned).
I think LOG_ERR should be kept in this case.